src/qtoolkit/io/sge.py

```diff
@@ -278,7 +291,10 @@ def _get_qstat_base_command(self) -> list[str]:
         return ["qstat", "-ext", "-urg", "-xml"]
 
     def _get_job_ids_flag(self, job_ids_str: str) -> str:
-        return f"-j {job_ids_str}"
+        raise NotImplementedError("Querying by job IDs is not supported for SGE.")
+
+    def _get_job_cmd(self, job_id: str):
+        raise NotImplementedError("Querying by job IDs is not supported for SGE.")
```

Comment:
I have added the error raise concerning the job_IDs at this two places in the code now.